modules/api/src/main/java/org/apache/ignite/lang/IgniteExceptionUtils.java

```diff
@@ -177,14 +177,15 @@ public static int extractCodeFrom(Throwable t) {
         return INTERNAL_ERR;
     }
 
-    // TODO: https://issues.apache.org/jira/browse/IGNITE-19539 this method should be removed or re-worked.
+    // TODO: This method should be removed or re-worked and usages should be changed to IgniteExceptionMapperUtil.mapToPublicException.
```

Comment:
I don't think that it's valid to have todo without ignite link.

Reply:
Yep, you are right. Will create a ticket.